packages/react-components/priority-overflow/src/overflowManager.ts

```diff
@@ -241,8 +302,8 @@ export function createOverflowManager(): OverflowManager {
     invisibleItemQueue.remove(itemId);
 
     if (item.groupId) {
-      overflowGroups[item.groupId].visibleItemIds.delete(item.id);
-      overflowGroups[item.groupId].invisibleItemIds.delete(item.id);
+      groupManager.removeItem(item.id, item.groupId);
+      item.element.removeAttribute('data-overflow-group');
```

Comment:
let's also add `data-overflow-item=itemId` too for easier debugging

Also please this data attributes to the `consts.ts` file

Reply:
fixed